i18n/de/fotema.ftl

```diff
@@ -32,10 +32,13 @@ selfies-album = Selfies
 animated-album = Animiert
 
 # Title for album showing all folders.
-folders-album = Ordner
+folders-album = Alle Alben
```

Comment:
I can't help but think the translation you are replacing is closer to meaning of the original English text. Same for `folder-album` below. However, I am not a native German speaker so I'll defer to your choices.

Reply:
> Should it be closer to the English meaning or more to the meaning itself?

A good question! To be honest, I had thought translations should or would be literal translations of the English text. However, having said that I think that is my ignorance of foreign languages and how translation works shining through. It is probably more important to have translations be idiomatic to the native speakers of that language that adhering to the "correct" English version.

I trust that you've given what you think is the best translation and I'm happy with accepting that.

Thank you for your PR.